### DISCUSSION TITLE
Voice and Video is not correct

### DISCUSSION THREAD

**Discussion by msdz** [community/events#37] [Community]
Voice and Video is not correct

**Comment by Alpakash** [comment]
Track five is indeed out of sync. Main stage audio is used over "How to open source as a startup"

**Reply by msdz** [reply]
Yes, We forwarded it to China users to watch, and the audio and video were inconsistent.

**Comment by clstaudt** [comment]
Please fix the audio/video switchboard. ;)

**Comment by dividor** [comment]
Reallllyyy wanted to see this presentation, audio is incorrect. 

It's now looping, very 80's sampler. :)

**Reply by Alpakash** [reply]
Also looked forward to this one. Definitely going to watch it back afterwards (if possible).

**Comment by msdz** [comment]
Still in the loop, hope to fix as soon as possible

**Comment by smoliicek** [comment]
I only see Looped animation on "How to open source as a startup" :(

**Reply by smoliicek** [reply]
Still in the loop

**Comment by Bestra** [comment]
Someone on our end is looking into it, we'll post updates if we receive any ❤️

**Comment by msdz** [comment]
Seems to be ready for the next presentation
![image](https://user-images.githubusercontent.com/10508130/200922810-b8b157f8-8dca-48d2-bb47-a109bf6b8c83.png)

**Reply by Alpakash** [reply]
Looks good to me. Lets hope the audio is right too this time ;-)

**Reply by msdz** [reply]
Consistent audio and video, Great!❤️❤️❤️
